.github/workflows/server_deploy.yaml

```diff
@@ -0,0 +1,31 @@
+name: Webserver deploy
+
```

Comment:
Maybe from `master` only? People would steal Heroku from each other on multiple branches.

Reply:
but it is deployed there only if changes to django server are made. I think that it would be easier in bta version when we would like to test the api